File: python/leagueranking/helpers/resultprocessorhelper.py
#Static helper functions to get the team name and score from the input string
class ResultProcessorHelper:

    @staticmethod
    def get_score(input: str):
        if input.split(' ')[-1].isnumeric():
            return input.split(' ')[-1]
        else:
            return None

    @staticmethod
    def get_team_name(input: str):
        return input.rsplit(' ', 1)[0].strip()

File: c#/Helpers/ResultProcessorHelper.cs
namespace LeagueRanking.Helpers
{
    /// <summary>
    /// Static helper class
    /// </summary>
    internal static class ResultProcessorHelper
    {
        //Get the score from the input string - Last value after the space
        public static int GetScore(string str)
        {
            bool success = Int32.TryParse(str.Split(' ').Last(), out int value);

            if (success)
                return value;

            return 0;
        }

        //Get the team name from the input string
        public static string GetTeamName(string str)
        {
            return str[..str.LastIndexOf(' ')].Trim();
        }

        
    }
}
